subcommand_plugin_examples/c_subcommand_plugin_tutorial/README.md

```diff
@@ -0,0 +1,243 @@
+[template]: https://github.com/conda/conda-plugin-template/subcommand_plugin_examples/c_subcommand_plugin_tutorial/generate
+[pyproject.toml docs]: https://packaging.python.org/en/latest/tutorials/packaging-projects/#creating-pyproject-toml
+[entrypoints docs]: https://packaging.python.org/en/latest/specifications/entry-points/
+[licenses]: https://docs.conda.io/projects/conda/en/latest/dev-guide/plugin-api/index.html#a-note-on-licensing
+[pep 621]: https://peps.python.org/pep-0621/
+[pluggy docs]: https://pluggy.readthedocs.io/en/stable/index.html
+
+# Custom conda Subcommand Plugin Tutorial Written in C
+
+In this tutorial, we will create a new conda subcommand written in C that converts Celsius to Fahrenheit.
+
+To follow along with this guide, make sure you have the latest conda and conda-build installed:
+
+```bash
+(base) $ conda update conda conda-build
```

Comment:
```suggestion
$ conda update conda conda-build pip
```

Let's axe the current environment information. I don't believe that's important to include and could confuse some.

Also, let's add `pip` as a dependency considering we use it later on in the tutorial.